tests/api_tests/nodes/test_views.py

```diff
@@ -2787,9 +2787,13 @@ def prepare_mock_wb_response(
     }
 
     if len(files):
-        data = [dict(default_file, **each) for each in files]
+        data = map(
```

Comment:
I am having to think far too much about this. What is it supposed to be doing, precisely?